backend/tracim_backend/applications/content_todo/controller.py

```diff
@@ -172,10 +178,11 @@ def delete_todo(self, context, request: TracimRequest, hapic_data=None):
             current_user=request.current_user, session=request.dbsession, config=app_config,
         )
 
-        todo = None  # type: typing.Optional['Todo']
+        todo_content = None  # type: typing.Optional['Content']
```

Comment:
I think this line is useless.

Reply:
If I don't put this declaration, the `todo_content` variable is defined inside the `try` below (line 182)
However I need this variable out of the `try` int the `with` (line 192)

I'm not sure I can remove this line safely

Reply:
It is safe as if the content is not found, an exception is raised (line 187) so the code afterward is not executed.